controllers/snapshot/snapshot_adapter.go

```diff
@@ -306,6 +307,27 @@ func (a *Adapter) EnsureCreationOfEphemeralEnvironments() (controller.OperationR
 					"integrationTestScenario.Name", integrationTestScenario.Name)
 				return controller.StopProcessing()
 			}
+			if clienterrors.IsInvalid(err) {
+
+				//a.logger.Error(err, "environment or deploymentTargetClass referenced by integrationTestScenario is invalid", "integrationTestScenario.Name", integrationTestScenario.Name)
```

Comment:
This can be removed

Reply:
removed in [b674ec](https://github.com/redhat-appstudio/integration-service/pull/421/commits/b674ecd242f54c9700dd14f99c04946a91f6af72)